client/data/authorizations/test/authorizations.fixture.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+      "created": "Sep 7, 2022 / 11:00AM",
+      "captured": false,
+      "order_id": 254,
+      "risk_level": "elevated",
```

Comment:
fyi, I've seen that the `risk_level` is marked as integer on the DB. I made some adjustments on my PR to address that, hopefully we can do the same here easily 🤞

Reply:
Good catch! I fixed it here [2028ea7](https://github.com/Automattic/woocommerce-payments/pull/4961/commits/2028ea7057dfd02686fe73ab2f3839e2914a80f0)